packages/firebase_crashlytics/firebase_crashlytics_platform_interface/lib/src/method_channel/method_channel_crashlytics.dart

```diff
@@ -87,9 +92,32 @@ class MethodChannelFirebaseCrashlytics extends FirebaseCrashlyticsPlatform {
     required String exception,
     required String information,
     required String reason,
+    bool fatal = false,
     List<Map<String, String>>? stackTraceElements,
   }) async {
     try {
+      if (fatal) {
+        try {
+          num currentUnixTimeSeconds =
+              (DateTime.now().millisecondsSinceEpoch / 1000).floorToDouble();
```

Comment:
```suggestion
          num currentUnixTimeSeconds =
              (DateTime.now().millisecondsSinceEpoch / 1000).floor();
```

Floor makes the most sense to me given that it should be seconds, so a whole number. And based on your screenshot on the issue if it's not showing the keys then it must be picking it up correctly and hiding them as they're internal keys.